commercial/app/views/contentapi/itemV2.scala.html

```diff
@@ -8,11 +8,55 @@
   optCapiReadMoreText: Option[String],
   optSponsorType: Option[String],
   optSponsorLabel: Option[String],
-  optClickMacro: Option[String],
-  optOmnitureId: Option[String])(implicit request: RequestHeader)
+  clickMacro: Option[String],
+  omnitureId: Option[String])(implicit request: RequestHeader)
```

Comment:
The `data-link-name` on the whole component was missing, which is where this parameter is used. I've now updated the wrapper to take it into account.